src/web/src/components/Search/List/SearchActionIcon.js

```diff
@@ -7,7 +7,12 @@ const SearchActionIcon = ({ search, loading, onRemove, onStop,...props }) => {
     return (<Icon name='spinner' loading {...props}/>);
   }
 
-  if (search.state.includes('Completed')) {
+  if (search.state.includes('Completed')
```

Comment:
the state flags end up being serialized as a comma-separated list, so this value will be something like `Completed, Cancelled`.  the state is guaranteed to contain the `Completed` flag when it's done, so we shouldn't need to include the different sub-states here